izanami-frontend/src/utils/queries.tsx

```diff
@@ -1231,14 +1231,39 @@ export function importUsersFile(tenant: string, file: FileList): Promise<any> {
   });
 }
 
-export function searchEntities(query: string): Promise<SearchResult[]> {
-  return handleFetchJsonResponse(fetch(`/api/admin/search?query=${query}`));
+export function searchEntities(
+  query: string,
+  filter: string[] = []
+): Promise<SearchResult[]> {
+  if (filter.length > 0) {
```

Comment:
To avoid duplicating too much code (here the url and the call to handleFetchJsonResponse) I think your if statement should value filterString if filters are present, else leave it to a blank string, for instance:

```js
let filterString = "";
   if (filter.length > 0) {
       filterString = filter.map(encodeURIComponent).join("&filter=");
   }
   // Perform your call here without needing to repeat anything.
```